Tactics/CSE.lean

```diff
@@ -104,6 +104,13 @@ structure CSEConfig where
   minOccsToCSE : Nat := 2
   /-- Whether the tactic should throw an error if no CSEable subterms were found. -/
   failIfUnchanged : Bool := true
+  /-- Number of steps the tactic should spend searching for subterms to gather information. -/
+  fuelSearch : Nat := 1000
+  /-- Number of steps the tactic should spend performing subexpression elimination. -/
+  fuelEliminate : Nat := 1000
```

Comment:
Minor nit: I think `searchFuel` and `eliminateFuel` would read more naturally.
to me `hasFuelSearch` communicates searching for fuel, while
`hasSearchFuel` communicates having fuel for searching (which is the intended meaning).

Very subjective, though, so feel free to ignore

Reply:
Changed accordingly.